hawkrest/management/commands/hawkrequest.py

```diff
@@ -1,47 +1,86 @@
-from optparse import make_option
+import logging
 
 from django.core.management.base import BaseCommand, CommandError
-
-import logging
 from mohawk import Sender
 
-from hawkrest import lookup_credentials
+from hawkrest import HawkAuthentication
+
+
+hawk_log = logging.getLogger('mohawk')
+hawk_log.setLevel(logging.DEBUG)
```

Comment:
This shouldn't execute at import time because debug logging might get turned on through regular usage of the `hawrest` middleware which is not what we want. It should only get turned on when the management command script runs so it should move back to the `handle()` function. 

Maybe it would help to add a comment to this code. I'd suggest something like: This will configure the `mohawk` library for debug logging so you can see inputs to signature functions and other useful stuff.

Reply:
Makes sense, I added it back.